examples/sol-anchor-contract/.gitignore

```diff
@@ -1,9 +0,0 @@
-yarn.lock
-Cargo.lock
```

Comment:
I think some of this items should probably stay

Reply:
These changes have been updated as requested.